ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryCleanupService.java

```diff
@@ -157,7 +160,7 @@ public void run() {
             SchemaRegistryUtil.cleanupInternalTopicSchemas(
                 queryTopicPrefix,
                 serviceContext.getSchemaRegistryClient(),
-                isTransient);
+                isTransient || isPersistent);
```

Comment:
is there ever a time when `isTransient || isPersistent` would be `false`? If not, can we just always pass in `true` here (or better yet, remove the flag altogether)?

Reply:
I had the same idea but was not sure whether it is always false!